README.md

```diff
@@ -0,0 +1,43 @@
+# Overview
+
+The Pentaho BI server is an open source, Apache licensed, business intelligence
+server written by Hitachi. It is fully featured, customisable and has a myriad
+of plugins available for it.
+
+This charm allows deployment of the Pentaho BI server to different environments
+and contains the Pentaho BI server and dedicated MySQL server for its management
+databases.
+
+# Usage
+
+To deploy the charm run:
+
+    juju deploy ~spiculecharms/pentaho-bi-server
+
+to make the BI server available you then need to run:
+
+    juju expose pentaho-bi-server
+
+You can then browse to http://ip-address:8080/pentaho to configure the service.
```

Comment:
Are you configuring the service or should it say server?

Reply:
Hmmm in juju parlance is should probably be Application